xml/System.Net.Http/HttpClient.xml

```diff
@@ -28,6 +28,10 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
+
+ > [!WARNING]
+ > <xref:System.Net.Http.HttpClient> is intended to be instantiated once and re-used throughout the life of an application. Instantiating an HttpClient class for every request will exhaust the number of sockets available under heavy loads. This will result in SocketException errors.
```

Comment:
```suggestion
 > <xref:System.Net.Http.HttpClient> is intended to be instantiated once and re-used throughout the life of an application. Instantiating an HttpClient class for every request will exhaust the number of sockets available under heavy loads. This will result in a <xref:System.Net.Sockets.SocketException>.
```